scriptmodules/libretrocores/lr-mame2003.sh

```diff
@@ -67,12 +67,12 @@ function configure_lr-mame2003() {
     # copy hiscore.dat and cheat.dat
     cp "$md_inst/metadata/"{hiscore.dat,cheat.dat} "$biosdir/$dir_name/"
     chown $user:$user "$biosdir/$dir_name/"{hiscore.dat,cheat.dat}
-    
+
     # lr-mame2003-plus also has an artwork folder
     if [[ "$md_id" == "lr-mame2003-plus" ]]; then
-        mkRomDir "$biosdir/$dir_name/artwork"
+        mkUserDir "$biosdir/$dir_name"/artwork
         cp "$md_inst"/metadata/artwork/* "$biosdir/$dir_name"/artwork/
-        chown -R $user:$user "$biosdir/$dir_name/artwork"
+        chown -R $user:$user "$biosdir/$dir_name"/artwork
```

Comment:
BTW this isn't actually how we generally style things - the previous line is.

Reply:
the cp line is also not in line with other code for the same reason. It's not a major issue, but just saying.

Reply:
no problem, will revert it ..  I like to minimise the quotes scope to where is really necessary :)

Reply:
I don't think there is a right or wrong here, but it's not how we do it elsewhere so better to stick with what we use currently.

Reply:
ok should be more like the other scriptmodules now... I gave instructions in the forum to test the PR, so when some feedback comes from the users you can merge it

Reply:
Thinking it better, maybe this PR should be merged ASAP and adjusted later if necessary (unlikely). Users are getting their romdirs polluted if they use the current version:

> Now there is a /home/pi/RetroPie/roms/home/pi/RetroPie/BIOS/mame2003-plus/artwork path on my Pi.